src/main/kotlin/day16/repository/ImageRepositoryImpl.kt

```
@@ -0,0 +1,71 @@
+            dataSource.saveImage(dataSource.fetchImage(url), path)
+        } catch (e: Exception) {
+            throw e
+        }
+
+        val endTime = System.currentTimeMillis()
+        val elapsedTime = endTime - startTime // 경과 시간
+
+        // 다운로드한 파일의 크기를 계산
+        val fileSize = try {
+            File(path).length().toString()
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)